src/server.js

```diff
@@ -22,11 +24,28 @@ const connected = false;
 const app = express();
 app.use(express.static(`${__dirname}/public`));
 
+// Setup exception logging
+const applicationMode = process.env.ENVIRONMENT;
+if (applicationMode === 'production' || applicationMode === 'staging') {
+    Sentry.init({
+        dsn: 'https://8ae89827146d49309d5a8e5699a605b6@sentry.io/1419602',
+        environment: applicationMode
+    });
+}
+
 // Redirect default '/' call to main.htm page
 app.get('/', (req, res) => {
     res.redirect('/index.html');
 });
 
+console.log("Start waiting....");
+if (process.env.IS_BACKUP === "true") {
```

Comment:
Please consider turning this into a ternary statement, i.e. `sleep.sleep(process.env.IS_BACKUP === "true" ? 10 : 15)`.

Reply:
That is way harder to read.

Reply:
Disagreed. Our code conventions also state that ‘else’ is not allowed, a ternary statement is the perfect solution in this scenario.

Reply:
Either way, these changes are:
- Not mine,
- Already in the master branch

Reply:
Furthermore, our code conventions state that 'else' SHOULD be avoided. Not that 'else' is not allowed.

Reply:
`sleep.sleep(process.env.IS_BACKUP === "true" ? 10 : 15)`
Does not improve readability in my opinion and an else is justified in this situation.

Also this code was already present in master and really too small to have this amount of discussion

Reply:
> ELSE block should be avoided whenever possible, use logic inversion or early return instead.

It was possible to avoid, so it should. Yes, this was pulled from `master` but that does not mean it can never be changed. We are team **REFACTOR**

Reply:
@HugovandeVliert Looking at the given `likes`, `dislikes` and comments, the majority is in favor of my comment. Please modify the code accordingly.

Reply:
done ae8ccf1.